cli/cli.ts

```diff
@@ -3066,7 +3066,15 @@ function buildCoreAsync(mode: BuildOption) {
                     console.log("no deploy functionality defined by this target")
                     return null;
                 }
-                return pxt.commands.deployCoreAsync(res);
+                return pxt.commands.deployCoreAsync(res)
```

Comment:
I think deploy should not fail when board are not available. having to add catch statements is just going to lead to messy code.